prac_09/cleanup_files.py

```diff
@@ -0,0 +1,81 @@
+"""
+CP1404/CP5632 Practical
+cleanup_files
+"""
+import shutil
+import os
+
+
+def main():
+    """Demo os module functions."""
+    print("Starting directory is: {}".format(os.getcwd()))
+
+    # Change to desired directory
+    os.chdir('Lyrics/Christmas')
+
+    # Print a list of all files in current directory
+    print("Files in {}:\n{}\n".format(os.getcwd(), os.listdir('.')))
+
+    # Make a new directory
+    # The next time you run this, it will crash if the directory exists
+    try:
+        os.mkdir('temp')
+    except FileExistsError:
+        pass
+
+    # Loop through each file in the (current) directory
+    for filename in os.listdir('.'):
+        # Ignore directories, just process files
+        if os.path.isdir(filename):
+            continue
+
+        new_name = get_fixed_filename(filename)
+        print("Renaming {} to {}".format(filename, new_name))
+
+        # Option 1: rename file to new name - in place
+        # os.rename(filename, new_name)
+
+        # Option 2: move file to new place, with new name
+        shutil.move(filename, 'temp/' + new_name)
+
+
+def get_fixed_filename(filename):
+    """Return a 'fixed' version of filename."""
+    new_name = filename.replace(" ", "_").replace(".TXT", ".txt")
+    temp = []
```

Comment:
What is "temp" and why is it a list (doesn't sound plural)?
Can you use a string instead of a list?